test/test_gpu2.cpp

```diff
@@ -1070,7 +1070,7 @@ TEST_F(NVFuserTest, FusionInputsIdLookup_CUDA) {
   nvfuser::InputsIdLookup inputs_id_lookup(2);
 
   // testing basic function, same encoding for identical inputs
-  auto id_0 = inputs_id_lookup.lookupId({t0, t1, 5.0});
+  auto id_0 = inputs_id_lookup.lookupId({t0, t1, 2.5});
```

Comment:
@kevinstephano FYI `lookupId` now uses all scalar values to create ID by default. This is because with dynamic shapes, any scalar value could potentially affect the structure of the concretized Fusion, and we cannot know which ones do affect it until the Fusion is built. See below in this test for example of masking scalar args out with the `record_scalar` vector argument.

Reply:
Nevermind. I actually updated this to default back to the behavior assumed here, so there should be no change in behavior for user schedules.